web/app/scripts/src/providers/project-config.es.js

```diff
@@ -0,0 +1,22 @@
+const ProjectConfig = function () {
+    let settings = {
+        apiRoot: 'localhost:8000'
+    };
+
+    return {
+        $get: function () {
+            return {
+                getApiRoot: () => settings.apiRoot
+            }
+        },
+        setApiRoot: (value) => {
+            settings.apiRoot = value;
+        }
+    };
+}
+
+export const provider = [
```

Comment:
After discussing in Slack, we can register modules and components in each file, then export the string identifiers so they can be used as dependencies in a DRY manner.

```js
// project-config.js

// Define imports and dependencies near each other.
import {SomeModuleName, SomeServiceName} from 'something.js';

const dependencies = [SomeModuleName];

// Define module code.
const ProjectConfig = function () {
    // ...
};

// Define identifier strings.
export const provider = 'ProjectConfig';

export const module = 'incuna-surveys.project-config';

// Register everything with Angular.
angular.module(module, dependencies)
    .provider(provider, ProjectConfig)
    .run([
        SomeServiceName,
        function (SomeService) {
            // Do something in the run phase with SomeService
        }
    ]);

// Export the module name by default.
export default module;
```

```js
// main.js
import ProjectConfigModule from 'project-config.js';

angular.module('incuna-surveys', [
    ProjectConfigModule,
    // ...
]);
```